files/scripts/jupyter_migration.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+
+# We need to list all the existing volumes related to Jupyter, usually named pvc-"[Aa0-zZ9]?-*", and we have to rule out the one for the deployment.
+# Then we should create snapshots for those volumens 
+# kubectl get persistentvolume --kubeconfig fauziv1/kubeconfig-KUBE_AWS -o name |cut -d/ -f2 |grep -v pvc-5a172428-d643-11e8-82ba-0e3bbdfa5f60 |while read i; do aws ec2 create-snapshot --description "SS-${i}" --tag-specifications "ResourceType=snapshot,Tags=[{Key=Name,Value=${i}}]" --volume-id $(kubectl --kubeconfig fauziv1/kubeconfig-KUBE_AWS get persistentvolume -o yaml --export $i |yq -r .spec.awsElasticBlockStore.volumeID | awk -F/ '{print $NF}'); done
+
+# kubectl get persistentvolume -o name |cut -d/ -f2 |grep -v pvc-47b422cc-854a-11e8-84de-0ec14f5e5db4 |while read i; do aws ec2 create-snapshot --description "SS-${i}" --tag-specifications "ResourceType=snapshot,Tags=[{Key=Name,Value=${i}}]" --volume-id $(kubectl get persistentvolume -o yaml --export $i |yq -r .spec.awsElasticBlockStore.volumeID | awk -F/ '{print $NF}'); done
+
+AZs=("a" "c" "d")
```

Comment:
why are AZs hard coded to a, c, d?

Reply:
The script is not something that you would run and it'll migrate the persistent volumens on it's own, it requires some manual intervention prior running and most likely some minor tweaks.

I'll try to find out a way to fully automate this, but if otherwise, the whole idea is not to run this blindly.